main/src/cgeo/geocaching/utils/Formatter.java

```diff
@@ -201,7 +201,7 @@ public static String formatWaypointInfo(final Waypoint waypoint) {
         if (waypointType != WaypointType.OWN && waypointType != null) {
             infos.add(waypointType.getL10n());
         }
-        if (Waypoint.PREFIX_OWN.equalsIgnoreCase(waypoint.getPrefix())) {
+        if (waypoint.getPrefix().toUpperCase().startsWith(Waypoint.PREFIX_OWN.toUpperCase())) {
```

Comment:
I feel not too comfortable with assuming that getPrefix() never returns null. Could you establish that by adding the necessary NonNull annotations?